orderable/tests/run.py

```diff
@@ -1,5 +1,5 @@
-import sys
 from argparse import ArgumentParser
```

Comment:
I'm not enamoured with this change, TBH, but it's fairly inconsequential.

I'll get this going now, and replace `flake8-import-order` with `isort` at another point.